addons/obs_websocket_gd/obs_websocket.gd

```diff
@@ -163,6 +165,10 @@ func _on_data_received() -> void:
 	if typeof(json_response) != TYPE_DICTIONARY:
 		print("Invalid json_response: %s" % json_response)
 		return
+		
+	if json_response.has("error"):
+		print("Error: %s" % json_response["error"])
+		emit_signal("obs_error", json_response["error"])
```

Comment:
Hmm. I'm not sure. If we do that are we basically treating every error as fatal?

Reply:
Yes. I guess it wouldn't hurt to also broadcast the entire error json in the `obs_updated`. At this point, I'm good with either approach.

Reply:
Since we already decided to let the client handle the errors I think it's probably safest to allow the obs_updated signal to occur, but let me go over the code a bit more and see if I can see a possible solution that I didn't see before.

Reply:
Okay so we may not need to create a new signal like this. The client already is receiving the `obs_update` signal which would include the error the client really could just handle it in it's `_on_obs_updated` right?

I may have just created extra work haha.

Let me try this out and I'll  get back with you.

Reply:
Okay I figured out a better way to do it using the signal we already had. That way the user doesn't have to worry about different types of signals.